cfg/conf.sample.php

```diff
@@ -33,7 +33,7 @@
 ; (optional) set a syntax highlighting theme, as found in css/prettify/
 ; syntaxhighlightingtheme = "sons-of-obsidian"
 
-; size limit per paste or comment in bytes, defaults to 10 Mebibytes
+; size limit per paste or comment in bytes, defaults to 10 Megabytes
 sizelimit = 10485760
```

Comment:
I [double-checked the number here too](https://duckduckgo.com/?q=10+mib+in+byte&t=ffab&ia=answer) and it is the correct value for 10 MiB.